src/components/pager/__internal__/pager-navigation.component.js

```diff
@@ -122,16 +146,26 @@ PagerNavigation.propTypes = {
   pageCount: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
   /** Sets the current page being shown */
   setCurrentPage: PropTypes.func,
-  /** onFirst Callback trigered when first link is triggered */
+  /** onFirst Callback triggered when first link is triggered */
   onFirst: PropTypes.func,
-  /** onPrevious Callback trigered when previous link is triggered */
+  /** onPrevious Callback triggered when previous link is triggered */
   onPrevious: PropTypes.func,
-  /** onNext Callback trigered when next link is triggered */
+  /** onNext Callback triggered when next link is triggered */
   onNext: PropTypes.func,
-  /** onLast Callback trigered when last link is triggered */
+  /** onLast Callback triggered when last link is triggered */
   onLast: PropTypes.func,
-  /** onPagination Callback trigered when a change is triggered */
+  /** onPagination Callback triggered when a change is triggered */
   onPagination: PropTypes.func,
+  /** Should the `First` navigation button be shown */
+  showFirstButton: PropTypes.bool,
```

Comment:
I'm thinking maybe group the `first` and `last` button props together, and group the `next` and `previous` button props together - I can't see a use case where you'd want `first` but not `last` etc and it would cut down the number of props/some complexity here

Reply:
See comment below